shadowsocks-csharp/Data/NLog.config

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<!-- configuration may reset after shadowsocks upgrade -->

```

Comment:
@studentmain here is the NLog config file. Allow user to modify some of the part but not encouraged. Once the code is merged, corresponding Wiki page would be created.

Reply:
Ok, if user broke the config, we can say "We warned you". It won't be problem now.